.github/workflows/rust.yml

```diff
@@ -226,3 +226,48 @@ jobs:
       - name: cargo nextest run --locked
         run: |
           cargo -Zgitoxide -Zgit nextest run --locked
+
+  # This job checks all crates individually, including no_std and other featureless builds
+  cargo-check-individually:
```

Comment:
technically not required if we can just clippy consensus runtime and domain runtimes with no-std